packages/core-v2/src/visual-model/entity-model/entity-model.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, EntityIdentifier } from "./entity";
+import { TypedObject } from "./typed-object";
+
+/**
+ * Represents a collection of entities with simple read access.
+ * By design this model is not provided with additional functionality or being extended.
+ * Instead a new functionality is added in form of separate interfaces.
+ * The ideas is for the consumer to define custom interface and thus declaring the required functionality.
+ */
+export interface EntityModel extends TypedObject {
+
+  /**
+   * @returns Model identifier.
+   */
+  getIdentifier(): string;
+
+  /**
+   * @returns Entity with given identifier.
+   */
+  getEntity(identifier: EntityIdentifier): Promise<Entity | null>;
```

Comment:
Maybe also provide `getEntities(): Promise<Entity[] | null>`? But maybe there is a good reason for not providing it here, for example the model is too big, etc.

Reply:
Exactly as you said. That is why we moved this functionality into `ListableEntityModel`.